packages/esm-outpatient-app/src/patient-search/empty-data-illustration.component.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+const EmptyDataIllustration = ({ width = '61', height = '59' }) => {
+  return (
+    <svg width={width} height={height} viewBox="0 0 61 59">
```

Comment:
Hey @CynthiaKamau, can you extract this `svg` as a separate file and then display it using `<img>` tag?

Embedding SVG images using `<img>` tag has few added benefits like browser caching, `alt` and `title` tag support, etc.

You can read more on that here - https://vecta.io/blog/best-way-to-embed-svg